pkg/ui/workspaces/db-console/src/views/cluster/containers/nodeGraphs/dashboards/networking.tsx

```diff
@@ -31,7 +30,7 @@ export default function (props: GraphDashboardProps) {
           <Metric
             name="cr.node.sys.host.net.recv.bytes"
             title={nodeDisplayName(nodeDisplayNameByID, nid)}
-            sources={storeIDsForNode(storeIDsByNodeID, nid)}
+            sources={nodeSources}
```

Comment:
Should this be `nid`? I think `nodeSources` is all the nodes or the selected node from the dropdown.

Reply:
Thanks for correcting, fixed.